app/views/assessments/index.html.erb

```diff
@@ -8,6 +8,16 @@
           return false;  // prevent click propagation
         });
       });
+      $.getJSON('metrics/get_num_new_instances',function(data, status){
+        if(status=='success' && data['num_new'] != 0){
+          // when there is a non-zero number of new instances
+          var new_instances_str = `${data["num_new"]} new`;
+          $('.new.badge.metrics-badge').attr('data-badge-caption', new_instances_str);
+        } else {
+          // otherwise remove the notification badge
+          $('.row.metrics').empty().append("<i class='material-icons left'>assessment</i> Metrics");
+        }
+      });
```

Comment:
as the this function has to wait for the ajax call to complete before removing the notification badge, if there are no new watchlist instances, the "new" button appears momentarily before disappearing. 

Would it be possible to add the badge only if there is non-zero? This is important as most people will start out not using the metrics feature yet.

Reply:
Good catch - just pushed a fix for it!